express-post-json/server.js

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+
+const app = express();
+
+const nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res, next) => {
+  res.json(Object.values(grades));
+  next();
+});
+
+app.use(express.json());
```

Comment:
Usually this would go at the top of the file. It's OK to keep it here, but since it can be applied to every route, it's good to put it before all the routes.